website/src/plugins/docusaurus-plugin-redirects/src/index.js

```diff
@@ -31,18 +31,13 @@ module.exports = function(context, opts) {
 <html lang="en-US">
   <head>
     <meta charset="UTF-8">
-    <script>
-      const redirectLink = '${newLink}' + location.search + location.hash;
-      document.write('<link rel="canonical" href="' + redirectLink + '">');
-      document.write('<title>Redirecting to ' + redirectLink + '</title>');
-      document.write('</head>')
-      document.write('<body>')
-      document.write('If you are not redirected automatically, follow this <a href="' + redirectLink + '">link</a>.')
-      document.write('</body>')
-      setTimeout(() => {
-        window.location.assign(redirectLink)
-      })
-    </script>
+    <meta http-equiv="refresh" content="0; url=${newLink}">
+    <link rel="canonical" href="${newLink}" />
+    <title>Redirecting to ${newLink}</title>
+  </head>
+  <script>
+    window.location.href = '${newLink}';
+  </script>
```

Comment:
The template here is a hybrid of the old one and the one provided by [@docusaurus/plugin-client-redirects](https://v2.docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects/) ([see the template here](https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-plugin-client-redirects/src/templates/redirectPage.template.html.ts)). The reason we can't use that instead is because it does not generate files the same way. The new plugin creates `index.html` nested within folders, so instead of generating a `replacement.html` page, it would create `replacement.html/index.html` which isn't what we want.